.golangci.yml

```diff
@@ -1,6 +1,9 @@
 ---
 run:
   timeout: 5m
+  skip-dirs:
```

Comment:
Why are you skipping these folders? I would lint at least our files.

Reply:
This was the quickest solution I've found to avoid Linter complaining about the geth code.

Reply:
let's review this when we decide to merge the release branch into the develop branch.